dockstore-webservice/src/test/java/core/WDLParseTest.java

```diff
@@ -130,6 +134,47 @@ private static SourceFile getSourceFile2() {
     }
 
 
+    /**
+     * Tests that Dockstore can handle a workflow with locally recursive imports
+     */
+    @Test
+    public void testLocallyRecursiveImport() {
+        String type = "workflow";
+        File recursiveWDL = new File(ResourceHelpers.resourceFilePath("local-recursive-import/localrecursive.wdl"));
+        //String primaryDescriptorFilePath = recursiveWDL.getAbsolutePath();
```

Comment:
If keeping this commented code in the PR I would add a note saying why it is there.

Reply:
fixed removed commented code